src/lib/output/components.ts

```diff
@@ -29,6 +29,11 @@ export abstract class ContextAwareRendererComponent extends RendererComponent {
     private location: string;
 
     /**
+     * Regular expression to test if a string looks like an external url.
+     */
+    protected urlPrefix: RegExp = /^(http|ftp)s?:\/\//;
```

Comment:
Do you think we should just do `/^\w{1,10}:\/\//` to ignore everything that looks absolute?

Reply:
I guess you're right here, I just wanted to PR a fix, changing to such a general thing might be a breaking change